src/plot/voronoi.js

```diff
@@ -56,7 +57,8 @@ Voronoi.defaultProps = {
   onMouseDown: NOOP,
   onMouseUp: NOOP,
   x: d => d.x,
-  y: d => d.y
+  y: d => d.y,
```

Comment:
might be better to have the default be something like:

```
cellClassName: d => d.className || ''
```

Reply:
Where would `d.className` come from? I can see that `d` is a polygon node from the voronoi instance but not clear on how classnames are attached to those.

Reply:
I see now that it comes from `d.data.className`, which is passed to the Voronoi as `nodes`.That makes me think, that `cellClassName` is unnecessary. I'll update this PR.